doc/api/events.md

```diff
@@ -1247,12 +1247,21 @@ target.addEventListener('foo', handler4, { once: true });
 ### `EventTarget` error handling
 
 When a registered event listener throws (or returns a Promise that rejects),
-by default the error is forwarded to the `process.on('error')` event
-on `process.nextTick()`. Throwing within an event listener will *not* stop
-the other registered handlers from being invoked.
+by default the error is treated as an uncaught exception on
+`process.nextTick()`. This means uncaught exceptions in `EventTarget`s will
+crash the Node.js process by default.
 
-The `EventTarget` does not implement any special default handling for
-`'error'` type events.
+Throwing within an event listener will *not* stop the other registered handlers
+from being invoked.
+
+The `EventTarget` does not implement any special default handling for `'error'`
+type events like `EventEmitter` does in order to be spec compliant.
+
+Currently errors are first forwarded to the `process.on('error')` event
+before reaching `process.on('uncaughtException')` - this behaviour will change
+in the next major release to align `EventTarget` with other Node.js APIs. Any
```

Comment:
I don't like naming "the next major release". I prefer if we say it is deprecated and will be removed in a future major release.

Reply:
Good idea, done

Reply:
@mcollina any chance this can get a second review? I'd prefer to make sure the changes are what you had in mind

Reply:
@mcollina ping (I'll also update it with Trott's suggestions before landing)